lib/clients.ex

```diff
@@ -6,33 +6,138 @@ defmodule GlassFactoryApi.Clients do
   alias GlassFactoryApi.Clients.Client
 
   @doc """
-  Get a client given the id
+  Return a tuple with `:ok` atom and the client from the specified ID
+  or an tuple with `:error` and a string with the error description.
 
   ## Examples
 
-      iex> GlassFactoryApi.Clients.get_client("1234")
-      %Client{
-        id: 1234,
-        name: "Google",
-        archived_at: null,
-        owner_id: 567,
-        office_id: 789
-      }
+  iex> GlassFactoryApi.Clients.get_client("1234")
```

Comment:
The code samples are inlined so that the documentation tools can syntax-highlight them correctly. Example from Elixir itself: https://github.com/elixir-lang/elixir/blob/055526057ac9992cc1df87441357c4e56d45235d/lib/elixir/lib/string.ex#L1186-L1200

Reply:
My editor alined the examples in a wrong way :(

I've done the suggestion here: https://github.com/Cendrao/glass_factory_api/pull/25/commits/b9602fb6001f00bc82a1f0f3907ca523cbf4d7ad

Thanks <3